cmd/colourlists.go

```diff
@@ -2,6 +2,7 @@ package cmd
 
 import (
 	"fmt"
+    "strconv"
```

Comment:
It does not look like the imports align properly, have you run `go fmt` on the file?

Reply:
done